app/auth/views/briefs.py

```diff
@@ -0,0 +1,121 @@
+from flask import abort, jsonify
+from flask_login import current_user
+from app.auth import auth
+from dmapiclient.audit import AuditTypes
+from ...models import db, AuditEvent, Brief, BriefResponse, Supplier, Framework
+from sqlalchemy.exc import IntegrityError, DataError
+from ...utils import (
+    get_json_from_request
+)
+
+
+@auth.route('/brief/<int:brief_id>', methods=["GET"])
+def get_brief(brief_id):
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+    brief_user_ids = [user.id for user in brief.users]
+    if current_user.role == 'buyer' and current_user.id in brief_user_ids:
+        return jsonify(brief.serialize(with_users=True))
+    else:
+        if brief.status == 'draft':
+            return abort(404)
```

Comment:
Do you want to set a developer friendly error message here as `jsonify(message='...`?

Reply:
Yeah I agree